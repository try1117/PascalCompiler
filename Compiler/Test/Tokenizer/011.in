1234567e-10