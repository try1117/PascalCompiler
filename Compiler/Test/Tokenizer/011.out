0  | 0  | CONST_DOUBLE             | 1234567e-10              | 0.000123456700000
